packages/core-entities/src/models/views/YieldViews.ts

```diff
@@ -587,6 +582,7 @@ export const YieldViews = (self: Instance<typeof NetworkModel>) => {
           underlying: t.underlying ? getTokenByID(t.underlying) : undefined,
```

Comment:
After making the change to look at getAllListedVaults, you can get the tvl and liquidity values from `t.vaultTVL`